src/Raskoh/Taxonomy.php

```diff
@@ -130,6 +130,7 @@ public function args()
             'show_admin_column' => $this->show_admin_column,
             'show_in_nav_menus' => $this->show_in_nav_menus,
             'show_tagcloud'     => $this->show_tagcloud,
+            'show_in_rest' => true,
```

Comment:
`show_in_rest` should not be hardcoded to `true` or `false`, can you make it configurable per instance basis?